.github/workflows/non-serverless-deploy.yml

```diff
@@ -34,3 +79,104 @@ jobs:
     name: Cypress Test
     uses: ./.github/workflows/cypress.yml
     secrets: inherit
+
+  slack-success:
+    needs:
+      - slack-started
+      - deploy-backend
+      - deploy-frontend
+      - deploy-worker
+      - e2e-test
+    if: success()
+    name: Send Slack message about successful build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: slackapi/slack-github-action@v1.23.0
+        with:
+          channel-id: C04H0FTGS5P # postman-deploys
```

Comment:
nit: Looks like Git Actions workflows supports "workflow level" environment variables. Since this value is used across multiple jobs does it make sense to define it once at the workflow level?
https://docs.github.com/en/actions/learn-github-actions/contexts#env-context